protocol/clproto_cpp/test/tests/include/encode_decode_helper.hpp

```diff
@@ -0,0 +1,41 @@
+#pragma once
```

Comment:
it's a nitpick on a nitpick but my written suggestion was to name the file after the function, `test_encode_decode.hpp`